### PR TITLE
[Please Merge] Added test PHP server to kubernetes cluster.

### PR DESCRIPTION
**Pull request by KugliLuca**
Added PHP server to kubernetes cluster

### CODE REVIEW

File: chart/templates/like-service.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        app.kubernetes.io/part-of: unguard
+    spec:
+      containers:
+        - name: {{.Values.likeService.name}}
+          image: {{.Values.likeService.deployment.container.image.repository}}:{{.Values.likeService.deployment.container.image.tag}}
+          imagePullPolicy: {{.Values.likeService.deployment.container.image.pullPolicy}}
+          ports:
+            - containerPort: {{.Values.likeService.deployment.container.ports.containerPort}}
+          env:
+            - name: ASPNETCORE_ENVIRONMENT
```
Comment (Dirli-V) [inline chart/templates/like-service.yaml]: [ISSUE] We don't need the env ASPNETCORE_ENVIRONMENT

File: chart/values.yaml [hunk 2]
```diff
@@ -374,6 +374,36 @@ proxyService:
...
+      - targetPort: 8000
+        port: 80
+
+  deployment:
+    strategy:
+      type: Recreate
+    container:
+      image:
+        repository: ghcr.io/dynatrace-oss/unguard/unguard-like-service
+        tag: 0.8.1
```
Comment (Dirli-V) [inline chart/values.yaml:393]: [DISCUSS] I think this tag is wrong. We need to set it to the actual version of our image. But I'm not 100% sure where this is coming from.

File: chart/values.yaml [hunk 3]
```diff
@@ -374,6 +374,36 @@ proxyService:
...
+        pullPolicy: IfNotPresent
+      ports:
+        containerPort: 8000
+      env:
+        JAEGER_AGENT_HOST: agent # change depending on your jaeger deployment
+        JAEGER_SAMPLER_TYPE: const
+        JAEGER_SAMPLER_PARAM: 0
+        JAEGER_DISABLED: true
+        ASPNETCORE_ENVIRONMENT: Production
```
Comment (Dirli-V) [inline chart/values.yaml]: [ISSUE] I think we don't need this

File: chart/values.yaml [hunk 4]
```diff
@@ -374,6 +374,36 @@ proxyService:
         JAEGER_SAMPLER_PARAM: 0
         OPENTRACING_JAEGER_ENABLED: false
 
+#like Service
```
Comment (Dirli-V) [inline chart/values.yaml]: [NIT]
```suggestion
# Like Service
```

File: src/like-service/config/app.php [hunk 5]
```diff
@@ -52,7 +52,7 @@
     |
     */
 
-    'url' => env('APP_URL', 'http://localhost'),
+    'url' => env('APP_URL', 'http://0.0.0.0:8000/like-service'),
```
Comment (Dirli-V) [inline src/like-service/config/app.php]: [DISCUSS] I think we should use the environment variables here to build the app url

File: src/like-service/config/database.php [hunk 6]
```diff
@@ -1,4 +1,4 @@
-<?php
+1<?php
```
Comment (Dirli-V) [inline src/like-service/config/database.php]: [ISSUE] 1 not needed :)

File: src/like-service/routes/api.php [hunk 7]
```diff
@@ -18,4 +18,6 @@
     return $request->user();
 });
 
+Route::
```
Comment (Dirli-V) [inline src/like-service/routes/api.php]: [ISSUE] Not needed

File: src/like-service/routes/web.php [hunk 8]
```diff
@@ -13,10 +13,10 @@
 |
 */
 
-Route::get('/', function () {
+Route::get('/like-service/', function () {
```
Comment (Dirli-V) [inline src/like-service/routes/web.php]: [DISCUSS] I think here we should also use the env vars (base path)

**Review by Dirli-V — CHANGES_REQUESTED** [review]

File: chart/templates/like-service.yaml (hunk 1 above)
Comment (KugliLuca) [reply chart/templates/like-service.yaml]: Removed

File: chart/values.yaml (hunk 2 above)
Comment (KugliLuca) [reply chart/values.yaml:393]: Discussed, not removed

File: chart/values.yaml (hunk 3 above)
Comment (KugliLuca) [reply chart/values.yaml]: Removed

File: chart/values.yaml (hunk 4 above)
Comment (KugliLuca) [reply chart/values.yaml]: Changed

File: src/like-service/config/app.php (hunk 5 above)
Comment (KugliLuca) [reply src/like-service/config/app.php]: enviroment variables are now used

File: src/like-service/config/database.php (hunk 6 above)
Comment (KugliLuca) [reply src/like-service/config/database.php]: removed

File: src/like-service/routes/api.php (hunk 7 above)
Comment (KugliLuca) [reply src/like-service/routes/api.php]: removed

File: src/like-service/routes/web.php (hunk 8 above)
Comment (KugliLuca) [reply src/like-service/routes/web.php]: env vars are now used

File: src/like-service/routes/api.php (hunk 7 above)
Comment (Dirli-V) [reply src/like-service/routes/api.php]: This is still here :)

Comment (KugliLuca) [reply src/like-service/routes/api.php]: Removed in separate commit

**Review by Dirli-V — APPROVED** [review]

**Pull request merged**